src/auth/index.js

```diff
@@ -8,7 +8,7 @@ const { JWT_CERT } = process.env;
 
 export { updateAppMetaData } from './auth0';
 
-export const namespace = 'https://www.freecodecamp.org/';
+export const namespace = 'https://auth-ns.freecodecamp.org/';
```

Comment:
Just curious and don't have too much context of Auth0, but why `auth-ns`?

Reply:
Its just a namespace that has to be any URL. For the purposes of keeping us sane, its short for auth namespace as `auth-ns`. There will never be an actual endpoint or anything here.

Reply:
This is kind of useful for huge orgs with lots of separate sub entities, like SSO, Windows AD, etc. where there will be a need for namespaces, but we dont't care about that for our purposes.